main.rb

```diff
@@ -0,0 +1,53 @@
+MORSE_MAP = {
+  '.-' => 'A',
+  '-...' => 'B',
+  '-.-.' => 'C',
+  '-..' => 'D',
+  '.' => 'E',
+  '..-.' => 'F',
+  '--.' => 'G',
+  '....' => 'H',
+  '..' => 'I',
+  '.---' => 'J',
+  '-.-' => 'K',
+  '.-..' => 'L',
+  '--' => 'M',
+  '-.' => 'N',
+  '---' => 'O',
+  '.--.' => 'P',
+  '--.-' => 'Q',
+  '.-.' => 'R',
+  '...' => 'S',
+  '-' => 'T',
+  '..-' => 'U',
+  '...-' => 'V',
+  '.--' => 'W',
+  '-..-' => 'X',
+  '-.--' => 'Y',
+  '--..' => 'Z'
+}.freeze
+
+def decode_char(character)
+  MORSE_MAP[character]
+end
+
+def decode_word(word)
+  msg_word = ''
+  characters = word.split
+  characters.each do |character|
+    msg_word += decode_char(character)
+  end
+  msg_word
+end
+decode_word('-- -.--')
+
+def decode(message)
+  msg = ''
+  words = message.split('   ')
+  words.each do |word|
+    msg += "#{decode_word(word)} "
+  end
+  print msg
+end
```

Comment:
- [ ] These methods returns `nil` at the end of the decoded morse code. You can fix this issue by removing the `print` statement on line `50` and then add `strip` to the msg returned from the method.  👉  `msg.strip` and the error will go away.

<img width="228" alt="chrome_vipZo802DD" src="https://user-images.githubusercontent.com/81468675/160832636-d73253f6-8704-43f5-b7e4-bcc10a994090.png">